libs/dh/metering-point/shell/src/test-setup.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import 'jest-preset-angular/setup-jest';
```

Comment:
issue: we must always as a minimum call `setUpTestbed` to ensure consistency in our tests

```suggestion
import 'jest-preset-angular/setup-jest';

import { setUpTestbed } from '@energinet-datahub/gf/test-util-staging';

setUpTestbed();

```